doc/build-instructions/Build_Instructions_V17.md

```diff
@@ -685,7 +685,7 @@ bash get_source.sh
 You must install a number of software dependencies to create a suitable build environment on your AArch64 Linux system:
 
 - GNU C/C++ compiler (The Docker image uses GCC 7.5)
-- [AArch64 Linux JDK](https://api.adoptopenjdk.net/v3/binary/latest/16/ga/linux/aarch64/jdk/openj9/normal/adoptopenjdk), which is used as the boot JDK.
+- [AArch64 Linux JDK](https://api.adoptopenjdk.net/v3/binary/latest/17/ga/linux/aarch64/jdk/openj9/normal/adoptopenjdk), which is used as the boot JDK.
```

Comment:
There are several other places that refer to a Java 16 bootjdk: please update them to Java 17.

Reply:
ah, ok. Done.